### DISCUSSION TITLE
how to link using mold?

### DISCUSSION THREAD

**Discussion by dzmitry-lahoda** [rust-cross/cargo-zigbuild#288] [Q&A]
We build via zigbuild static release binary with musl. It is slow.

We thinking to make build faster with https://github.com/rui314/mold .

How I can use mold?

**Comment by polarathene** [comment]
Mold will probably not make a difference. Did you try building without zigbuild, was that slow too and suddenly faster with mold?

If you have not verified such why do you think mold would make a difference?

What you'll find is the default memory allocator in musl is slow (notably at multi-thread workloads), builds can be 2-3x slower on musl to complete than with glibc cargo. A way to speed this up is to run cargo with `mimalloc` via `LD_PRELOAD` ENV.
